roxctl/scanner/generate/generate.go

```diff
@@ -108,6 +108,7 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 
 	c.Flags().StringVar(&scannerGenerateCmd.outputDir, "output-dir", "", "Output directory for scanner bundle (leave blank for default)")
 	c.Flags().StringVar(&scannerGenerateCmd.apiParams.ScannerImage, flags.FlagNameScannerImage, "", "Scanner image to use (leave blank to use server default)")
+	c.Flags().StringVar(&scannerGenerateCmd.apiParams.ScannerV4Image, flags.FlagNameScannerV4Image, "", "Scanner V4 image to use (leave blank to use server default)")
```

Comment:
Don't we need ScannerV4DB here too?

Reply:
Oh, thanks. 🤦

Reply:
Even though it wasn't clear to me why `ScannerImage` was there but not `ScannerDBImage`, I have added a flag for `SCannerV4DBImage`.
Didn't want to touch `ScannerDBImage` for now to prevent scope creep.